igvc_rviz_plugins/configs/jessii_conf.rviz

```diff
@@ -0,0 +1,289 @@
+Panels:
```

Comment:
We should have more of these but I am not sure how to best organize them. Opening up an issue for it

Reply:
So the reason i used this specific configuration is because it highlights everything the robot is capable of doing at the moment:
* sensing with the lidar
* creating an occupancy grid
* planning and following paths using the occupancy grid

If you want, i can remove it from this PR but I think it adds value as a simple debugging tool.

Reply:
No leave it in. I like it. I just mean later on we should organize them